### ISSUE TITLE
logfile gets written only after the first frame was rendered

### ISSUE THREAD

**Issue by chrisgraf** [boku-ilen/landscapelab#17]
we need the logfile already at startup of godot to debug the startup process

**Comment by kb173** [comment]
Perhaps we should open an issue at Godot for this?

**Comment by chrisgraf** [comment]
I think this could be worked around at our side by starting blocking tasks, like the server requests in individual threads

**Comment by chrisgraf** [comment]
maybe we can render this issue obsolete with my proposal in #195 .. so we start rendering the first frame before we begin the startup?

**Comment by kb173** [comment]
I think this is mostly a problem with Singletons - their loading process still can't be debugged because they load before any frame is rendered. We're trying to get rid of Singletons more and more, so this should become less of a problem. Overall, I'd say there's nothing we can do about it and no need to actively keep track of it, so I'd say we could close this.

**Comment by chrisgraf** [comment]
Not ideal but we probably have to wait for a better logging handling by godot. For the introduction of the updated logger I will open a new issue but this will not help with the issue at hand.
